caffe2/python/onnx/backend.py

```diff
@@ -173,12 +173,29 @@ class Caffe2Backend(Backend):
         'RNN': '_create_rnn',
     }
 
+    # DummyName factory
+    @classmethod
+    def __init__(cls):
```

Comment:
You can put `_dummy_name_factory ` as class scope variable to serve as class static variable:
```
class A(object):
  x = 1
  @classmethod
  def f(cls):
    print(cls.x)
```

Reply:
But I want to avoid `static`. :)

Reply:
But you decorate all these functions with `@classmethod`, so it's already static?

Reply:
It's avoidable, but this python Backend class is designed to be singleton (you can see all the functions are decorated as `@classmethod`). So whether it's a member variable or class static variable doesn't really matter :-)

Reply:
Yeah, I think I'll just use class scope variable instead.